src/wmtk/invariants/SmallerFunctionInvariant.cpp

```diff
@@ -0,0 +1,39 @@
+#include "SmallerFunctionInvariant.hpp"
+
+#include <wmtk/Mesh.hpp>
+#include <wmtk/function/PerSimplexFunction.hpp>
+
+namespace wmtk::invariants {
+
+SmallerFunctionInvariant::SmallerFunctionInvariant(
```

Comment:
Is that code you copied from the embedded remeshing branch? I know this code already existed somewhere but maybe it was not in the main branch.